src/rules/declaration-block-no-redundant-longhand-properties/README.md

```diff
@@ -0,0 +1,131 @@
+# declaration-block-no-redundant-longhand-properties
+
+Disallow longhand properties that can be combined into one.
+
+```css
+a {
+    padding-top: 1px;
+    padding-right: 1px;
+    padding-bottom: 1px;
+    padding-left: 1px; }
+/** ↑
+ *  These longhand properties */
+```
+
+The longhand properties in the example above can be more concisely written as:
+
+```css
+a {
+  padding: 1px;
+}
+```
+
```

Comment:
Please add a note explaining that this rule will only warn if you've used the longhand equivalent of *all* the properties that the shorthand will set, and why. (I'm sure users will file issues about this.)